js-core/homeworks/homework-9/src/main.js

```diff
@@ -0,0 +1,207 @@
+//Task1
+let developer1 = {
+  skills: ['JavaScript', 'linux', 'html', 'OOP', 'Node.js'],
+  requirements: ['Node.js', 'JavaScript', 'OOP'],
+  goodDev: goodDev
+};
+let developer2 = {
+  experience: [
+    { technology: 'java' },
+    { technology: 'c++' },
+    { technology: 'aws' },
+    { technology: 'docker' }
+  ],
+  requirements: ['java', 'json', 'c++', 'JavaScript'],
+  goodDev: goodDev
+};
+
+function goodDev(dev) {
+	let arrRequirements = this.requirements;
+	if(this.skills) {
+		let arrSkills = this.skills;
```

Comment:
arr is a type, it's not really good to have type information at variable naming.

developerSkills sound much better